diff --git a/build/modules.conf.in b/build/modules.conf.in
index 14ad5dd080..96268ca63e 100644
--- a/build/modules.conf.in
+++ b/build/modules.conf.in
@@ -1,5 +1,5 @@
 #applications/mod_abstraction
-applications/mod_av
+#applications/mod_av
 #applications/mod_avmd
 #applications/mod_bert
 #applications/mod_blacklist
@@ -8,7 +8,7 @@ applications/mod_av
 #applications/mod_cluechoo
 applications/mod_commands
 applications/mod_conference
-#applications/mod_curl
+applications/mod_curl
 #applications/mod_cv
 applications/mod_db
 #applications/mod_directory
@@ -71,7 +71,7 @@ codecs/mod_b64
 codecs/mod_g723_1
 codecs/mod_g729
 codecs/mod_h26x
-#codecs/mod_ilbc
+codecs/mod_ilbc
 #codecs/mod_isac
 #codecs/mod_mp4v
 codecs/mod_opus
@@ -155,14 +155,14 @@ say/mod_say_en
 #say/mod_say_nl
 #say/mod_say_pl
 #say/mod_say_pt
-#say/mod_say_ru
+say/mod_say_ru
 #say/mod_say_sv
 #say/mod_say_th
 #say/mod_say_zh
 #timers/mod_posix_timer
 #timers/mod_timerfd
 xml_int/mod_xml_cdr
-#xml_int/mod_xml_curl
+xml_int/mod_xml_curl
 #xml_int/mod_xml_ldap
 #xml_int/mod_xml_radius
 xml_int/mod_xml_rpc
